packages/pages/src/components/image/types.ts

```diff
@@ -34,29 +49,58 @@ export const ImageLayout = {
   FILL: "fill",
 } as const;
 
-export type ImageLayout = typeof ImageLayout[keyof typeof ImageLayout];
-
 /**
- * The shape of the data passed directly to the different template functions with the
- * exception of the render function (getPath, getHeadConfig, etc).
+ * The type definition for the image layout.
+ *
+ * @public
  */
-export type ImageProps = {
+export type ImageLayout =
+  typeof ImageLayoutOption[keyof typeof ImageLayoutOption];
+
+interface BaseImageProps {
   /** The image field from Knowledge Graph. */
   image: ImageType;
   /** Overrides the className on the underlying img tag. */
   className?: string;
+  /** Specifies how the image is rendered. */
+  layout?: ImageLayout;
   /** The absolute width of the image. Only impacts if layout is set to "fixed". */
   width?: number;
   /** The absolute height of the image. Only impacts if layout is set to "fixed". */
   height?: number;
   /** The aspect ratio of the image. Only impacts if layout is set to "aspect". */
   aspectRatio?: number;
-  /** Specifies how the image is rendered. */
-  layout?: ImageLayout;
   /** A pass through react component that is displayed when the image is loading. */
   placeholder?: React.ReactNode;
   /** Pass through props that are on the native HTML img tag. The Image component may not work if src and/or srcsets are included. */
   imgOverrides?: Object;
   /** The pass through style of the underlying img tag. */
   style?: React.CSSProperties;
-};
+}
+
+interface OtherImageProps extends BaseImageProps {
```

Comment:
jsdoc for all of these with @public

Reply:
I don't think we want to export these?

Reply:
Added docs but didn't export the layout specific interfaces. Let me know if you want to export them.